### DISCUSSION TITLE
Changing the percentage at which it calculates show as "watched"

### DISCUSSION THREAD

**Discussion by StarzeeBEAST** [mach64/trakt-scrobbler#252] [General]
Hi,

I've searched all over but cannot find any command line for this. Is it possible to do? I'd like to set it so the scrobbler records a file as "watched" at about 70% progress into the file.

Thanks.

**Comment by iamkroot** [comment]
Nope, sorry. [Trakt docs](https://trakt.docs.apiary.io/#reference/scrobble/stop) mandate that anything below 80% will count as a pause, and there's no way to change that as far as i know.

**Reply by StarzeeBEAST** [reply]
Ah, i see, didn't realise it was on their end. Well thank you anyway!
